src/components/Filters/index.js

```diff
@@ -330,15 +330,13 @@ class Filters extends React.Component {
           <div>
             <Modal
               title={
-                this.state.edit.id
-                  ? 'Update existing filter'
-                  : 'Add a new filter'
+                this.state.edit.id ? 'Update existing filter' : 'Filter by'
```

Comment:
Would also make sense to me to change `Update existing filter` to something simpler like `Update filter` or even use `Filter by` for both cases. What do you think @LucianBuzzo ?

Reply:
Lets use `Filter by` in both cases